include/Note.h

```diff
@@ -205,6 +205,12 @@ class LMMS_EXPORT Note : public SerializingObject
 		return classNodeName();
 	}
 
+	inline QString getNoteName() const
+	{
+        return m_nameList[m_key % 12] +
+               QString::number(static_cast<int>(m_key / KeysPerOctave));
+	}
```

Comment:
Let's have this return a `const char*` and then convert that to QString where needed.

Reply:
Done

Reply:
Sorry I wasn't more explicit on this. I meant entirely remove Qt references from it. So the `+ QString::number` too.

Reply:
Ideally we shouldn't have any Qt in this header.

Reply:
What about std:: ? because in C++ i'd more return an std::string outside of the context of Qt than const char*

Reply:
That's fine! Just trying to make sure we don't include Qt stuff in core, and Note is definitely a core thing. :)

Reply:
looking at the code from Note.h a lot of stuff seems to actually depend on Qt, the notevector is q QVector, the saveSettings and loadSettings depend on QDomDocument and QDomElement, classNodeName uses QString etc

it's because Note inherits from SerializingObject

Reply:
Ah crap, I didn't notice the save/load-Settings. I saw NoteVector, but that can easily be changed to a std::vector. We'll have to figure out another way later then. For now let's still try to keep as much Qt out as we can.

Reply:
I agree we should move away from Qt where possible, it can maybe hold when it makes things too complex (for example, on the AutomationPattern I still used a QMap because it's iterator is random access while `std::map` would require some extra code to simulate random access). But even if the class makes unnecessary use of Qt, when we can we should avoid it in new code, and possibly slowly remove those existing unnecessary uses through other PRs.

This is a case where not using Qt is a small change thankfully :smile:

Reply:
I changed it to use a more standard c++ way now